packages/shared/src/lib/core/profile/actions/active-profile/login.ts

```diff
@@ -94,7 +95,8 @@ export async function login(loginOptions?: ILoginOptions): Promise<void> {
          * create one for the new profile.
          */
         if (accounts?.length === 0) {
-            await createNewAccount()
+            const newAccount = await createNewAccount()
+            accounts.push(newAccount)
```

Comment:
Either
```suggestion
            accounts = [newAccount]
```

or initialize accounts in line 80 with `[ ]`